### PR TITLE
Revert "refactor: don't expose Deno[Deno.internal].core namespace"

### PR DESCRIPTION
**Pull request by bartlomieju**
Also conditionally disabled one test if there's not enough space on device.

### CODE REVIEW

File: cli/tests/unit/read_text_file_test.ts [hunk 1]
```diff
@@ -164,7 +164,13 @@ Deno.test(
...
     const filePath = "cli/tests/testdata/too_big_a_file.txt";
 
-    Deno.writeFileSync(filePath, bytes);
+    try {
+      Deno.writeFileSync(filePath, bytes);
+    } catch (e) {
+      // NOTE(bartlomieju): writing a 0.5Gb file might be too much for CI,
+      // so skip running if writing fails.
+      return;
+    }
```
Comment (dsherret) [inline cli/tests/unit/read_text_file_test.ts:173]: Maybe this should be a separate PR in case we revert the revert?

File: cli/tests/unit/read_text_file_test.ts [hunk 2]
```diff
@@ -164,7 +164,13 @@ Deno.test(
     const bytes = new Uint8Array(kStringMaxLengthPlusOne);
     const filePath = "cli/tests/testdata/too_big_a_file.txt";
 
-    Deno.writeFileSync(filePath, bytes);
+    try {
+      Deno.writeFileSync(filePath, bytes);
+    } catch (e) {
```
Comment (dsherret) [inline cli/tests/unit/read_text_file_test.ts:169]: Do we have a way to check the specific error instead of returning on all errors?

**Review by dsherret — APPROVED** [review]
LGTM

File: cli/tests/unit/read_text_file_test.ts (hunk 2 above)
Comment (bartlomieju) [reply cli/tests/unit/read_text_file_test.ts]: It's OS specific so I'm not sure what will the error be on other OSes.

File: cli/tests/unit/read_text_file_test.ts (hunk 1 above)
Comment (bartlomieju) [reply cli/tests/unit/read_text_file_test.ts:173]: I'd rather get it in here, instead of waiting for another PR to pass CI
